pkg/vm/engine/tae/logstore/driver/batchstoredriver/file.go

```diff
@@ -35,20 +35,20 @@ func MakeVersionFile(dir, name string, version uint64) string {
 	return fmt.Sprintf("%s-%d%s", filepath.Join(dir, name), version, suffix)
 }
 
-func ParseVersion(name, prefix, suffix string) (int, error) {
+func ParseVersion(name, prefix, suffix string) (n int, ok bool) {
```

Comment:
All file names are generated by us, why we should ever see a parse error?

Reply:
There're other files in the directory.

Reply:
Which directory.   What file.   Why they land in same directory.

This does not make sense.   We control where the file is.   If the file name encode a version or something, bad file should never reach here.   This check is important -- only to catch file/dir corruption.